### PR TITLE
feat: test selector - add chain id 3337

### PR DESCRIPTION
**Pull request by ajgrande924**

### CODE REVIEW

**Review by stackman27 — APPROVED** [review]
LGTM

File: test_selectors.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ selectors:
     selector: 11787463284727550157
   2337:
     selector: 12922642891491394802
+    name: geth-testnet-2
```
Comment (andrevmatos) [inline test_selectors.yml:7]: Should we call those instead?
```suggestion
    name: geth-devnet-2
```

Comment (stackman27) [reply test_selectors.yml:7]: @ajgrande924 thoughts?

Comment (ajgrande924) [reply test_selectors.yml:7]: that works lemme update

**Review by andrevmatos — APPROVED** [review]

**Pull request merged**